src/RestClient/endpoints/V3/Themes/ThemeCustomTemplates.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeCustomTemplate } from './types';
+
+class ThemeCustomTemplates {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Enumerates available custom templates for in the theme files in a specific Theme version
+   *
+   * @param versionUUID A valid store theme version UUID
+   * @returns Promise resolving to a list of available custom template files
+   */
+  get(versionUUID: string): ThemeCustomTemplate['GetResponse'] {
+    return this.client.post(`${themesPath}/custom-templates/${versionUUID}`);
```

Comment:
The method is named `get`, but the http client is invoking `post`. This needs to be changed to `get` according to our documentation: https://developer.bigcommerce.com/api-reference/1fea285a88538-get-custom-templates